www/src/components/clusters/ClustersSidecar.tsx

```diff
@@ -0,0 +1,61 @@
+import { A, Flex } from 'honorable'
+import moment from 'moment'
+import {
+  ArrowTopRightIcon, Button, Sidecar, SidecarItem,
+} from 'pluralsh-design-system'
+import { ReactElement, useContext } from 'react'
+
+import QueueContext from '../../contexts/QueueContext'
+
+// TODO: This should not be needed once Clusters.js file gets removed
+// @ts-ignore
+import { Queue } from './Clusters.tsx'
+
+export function ClustersSidecar(): ReactElement {
+  const queue: Queue = useContext(QueueContext)
+
+  return (
+    <Flex
+      gap={24}
+      direction="column"
+      paddingVertical="large"
+    >
+      <Button
+        secondary
+        endIcon={<ArrowTopRightIcon size={24} />}
+        as={A}
+        target="_blank"
+        href={`https://${queue.domain}`}
+        {...{
+          '&:hover': {
+            textDecoration: 'none',
+          },
+        }}
+      >
+        Console
+      </Button>
+      <Sidecar heading="Metadata">
+        <SidecarItem heading="Cluster name">
+          {queue.name}
+        </SidecarItem>
+        <SidecarItem heading="Git url">
```

Comment:
Should we switch to `URL`?

Reply:
Not sure. I have followed the design here.